satpy/readers/nc_olci.py

```diff
@@ -33,6 +33,7 @@
 from satpy.readers.file_handlers import BaseFileHandler
 from satpy.utils import angle2xyz, xyz2angle
 from satpy import CHUNK_SIZE
+from functools import reduce
```

Comment:
This has been fixed in #515 but has not been merged yet. Please remove it from this PR and we'll include it from that PR.